packages/react-charting/docs/TestPlans/VerticalStackedBarChart/ComponentTests.md

```diff
@@ -0,0 +1,73 @@
+|                         - Double mouse click on bar legends                         |                     Should deselect legend on double mouse click on respective legend                     |      RTL      |
+|                        - Single mouse click on line legends                         |                      Should select legend on single mouse click on respective legend                      |      RTL      |
+|                        - Double mouse click on line legends                         |                     Should deselect legend on double mouse click on respective legend                     |      RTL      |
+|                          Test 8: [Sub-Component]: Callout                           |                                                                                                           |               |
+|                              - Hover mouse over a bar                               |                                 Should call the handler on mouse over bar                                 |      RTL      |
+|                     - Hover mouse over a bar to display callout                     |                               Should show the default callout over that bar                               |      RTL      |
+|                          - Hover mouse over a stacked bar                           |                                 Should show the default stacked call out                                  |      RTL      |
+|                             - Hover mouse over the line                             |                              Should show the default callout over that line                               |      RTL      |
+|                 - Specify custom callout and hover mouse over a bar                 |                               Should show the custom callout over that bar                                |      RTL      |
+|               - Specify custom callout and hover mouse over the line                |                               Should show the custom callout over that line                               |      RTL      |
+|                                - Mouse click on bar                                 |                             Should call the handler on mouse click on the bar                             |      RTL      |
+|                       Test 9: [Sub-Component]: x-axis labels                        |                                                                                                           |               |
+|                              - Truncate x-axis labels                               |                            Should show the x-axis labels tooltip when hovered                             |      RTL      |
+|                               - Rotate x-axis labels                                |                               Should rotate the x-axis labels by 45 degrees                               |      RTL      |
```

Comment:
Add test for these 2 props:
bargapmax sets the maximum gap between bars in a stack. 
barCornerRadius sets the corner radius of the bars